packages/ui/src/bounty/modals/AddBountyModal/AddBountyModal.tsx

```diff
@@ -43,6 +45,11 @@ export const AddBountyModal = () => {
   const { allAccounts } = useMyAccounts()
   const [state, send, service] = useMachine(addBountyMachine)
   const [isValidNext, setValidNext] = useState(false)
+
+  const { refetch: refetchBounties } = useBounties({ status: 'active' })
+  useRefetch({ type: 'set', payload: refetchBounties })
+  useRefetch({ type: 'do', payload: state.matches(AddBountyStates.success) })
```

Comment:
I approved this way of re-fetching in the previous PR (because I didn't remember how some of these modals where called) but IMO it would be better to call the `useRefetch({ type: 'set', payload: refetch })` inside the components where the entities are listed (here it would be `packages/ui/src/app/pages/Bounty/components/BountiesLayout.tsx`). And just call: `useRefetch({ type: 'do', ... ` on the modals. This entities would be refetched with the correct filters/order in just 1 query.